integration/common.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package integration
+
+import (
+	"github.com/eclipse-kanto/kanto/integration/util"
+	"github.com/stretchr/testify/suite"
+)
+
+type uploadTestSuite struct {
+	suite.Suite
+
+	initializer *util.SuiteInitializer
```

Comment:
Don't use pointers here, this type contains pointers only and it's supposed to be used with embedding just like suite.Suite